src/svg-renderer.js

```diff
@@ -197,6 +199,30 @@ class SvgRenderer {
         }
     }
 
+    /**
+     * Fix SVGs to comply with SVG spec. Scratch 2 defaults to x2 = 0 when x2 is missing, but
+     * SVG defaults to x2 = 1 when missing.
+     */
+    _transformGradients () {
+        // Collect all gradient elements into a list.
+        const linearGradientElements = [];
+        const collectElements = domElement => {
+            if (domElement.localName === 'linearGradient') {
```

Comment:
Are there any similar issues with radial gradients?

Reply:
From my testing, Scratch 2 always sets cx, cy, and r on radial gradients, even when they're 0. It's also really hard or maybe impossible to get these values to be 0 using the paint editor in Scratch 2